kubevirt-vmware/deploy/crds/v2v.kubevirt.io_ovirtproviders_crd.yaml

```diff
@@ -0,0 +1,74 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+    name: ovirtproviders.v2v.kubevirt.io
+spec:
+    group: v2v.kubevirt.io
+    names:
+        kind: OVirtProvider
+        listKind: OVirtProviderList
+        plural: ovirtproviders
+        singular: ovirtprovider
+    scope: Namespaced
+    categories:
+        - all
+    subresources:
+        status: {}
+    validation:
+        openAPIV3Schema:
+            description: OVirtProvider is the Schema for the ovirtproviders API
+            properties:
+                apiVersion:
+                    description:
+                        "APIVersion defines the versioned schema of this representation
+                        of an object. Servers should convert recognized schemas to the latest
+                        internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources"
+                    type: string
+                kind:
+                    description:
+                        "Kind is a string value representing the REST resource this
+                        object represents. Servers may infer this from the endpoint the client
+                        submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds"
+                    type: string
+                metadata:
+                    type: object
+                spec:
+                    description: OVirtProviderSpec defines the desired state of OVirtProvider
+                    properties:
+                        connection:
+                            type: string
+                        timeToLive:
+                            type: string
+                        vms:
+                            items:
+                                description: OVirtVM aligns with maintained UI interface
+                                properties:
+                                    detail:
+                                        description: OVirtVMDetail contains ovirt vm details as json string
+                                        properties:
+                                            raw:
+                                                type: string
+                                        type: object
+                                    detailRequest:
+                                        type: boolean
+                                    name:
+                                        type: string
+                                required:
+                                    - name
```

Comment:
can we also include id, datacluster and cluster for better filtering and vm uniqueness? The API doesn't have to be the same as v2v vmware. The UI is quite flexible

Reply:
We have a cluster name in a secret and vm id is [part of the details ](https://github.com/ManageIQ/manageiq-v2v-conversion_host/pull/32/files/57354cb1027b42008421a692a357c573d2c89dab#diff-07147f3641032dee71e1db5e7a8999f5R87) as well. Do you think anything else is needed?

Reply:
The name is not unique accross the datacenter. So it would be nice to have all information needed for the user to choose the vm before we send detailRequest. I would imagine having Datacenter and Cluster name and id should be sufficient

Reply:
It is enough to use cluster and vm name is sufficient to uniquely identify a vm.